### PR TITLE
This PR simplifies PR#23 (Farm page feedback)

### PR DESCRIPTION
**Pull request by schystz**
Simplifies [PR#23](https://github.com/HaloDAO/halodao-interface/pull/23) by creating a new branch (`features/hdf-152-farm-page-feedback`) from latest `develop` branch, and merging `feature/hdf-101-farm-page` to it.

The goal is to lessen the things we see in Commits and Files changed tab, so it is easier to review.

### CODE REVIEW

File: src/components/PositionCard/BalancerPoolCard.tsx [hunk 1]
```diff
@@ -511,6 +584,26 @@ export default function BalancerPoolCard({ poolInfo, tokenPrice }: BalancerPoolC
                     </StyledRowFixed>
                   </StyledBalanceStakeWeb>
                 </StyledFixedHeightRowWeb>
+                <StyledRowFixed>
+                  <StyledExternalLink
+                    href="https://balancer.exchange/#/swap"
```
Comment (schystz) [inline src/components/PositionCard/BalancerPoolCard.tsx:589]: @barryhalo please use `poolInfo.balancerUrl` here (same with line 755). We want to take the user to balancer's add liquidity page not to swap page.

Comment (schystz) [inline src/components/PositionCard/BalancerPoolCard.tsx]: @barryhalo please use `poolInfo.balancerUrl` here (same with line 755). We want to take the user to balancer's add liquidity page not to swap page.

**Review by schystz — COMMENTED** [review]
Hi @barryhalo , I am skipping any feedback to stake and unstake button and taking it up myself instead. I only have 1 minor feedback below.

File: src/components/PositionCard/BalancerPoolCard.tsx (hunk 1 above)
Comment (barryhalo) [reply src/components/PositionCard/BalancerPoolCard.tsx]: Got it, checking

Comment (barryhalo) [reply src/components/PositionCard/BalancerPoolCard.tsx]: @schystz 

> Hi @barryhalo , I am skipping any feedback to stake and unstake button and taking it up myself instead. I only have 1 minor feedback below.

@schystz This is fixed already

**Comment by barryhalo** [comment]
> Hi @barryhalo , I am skipping any feedback to stake and unstake button and taking it up myself instead. I only have 1 minor feedback below.

@schystz This fixed already

**Comment by schystz** [comment]
HDF 154-157 looks good to me, so approving!

Edit: I've opened this PR so GH won't let me approve it :D
Consider +1 to the approve count @barryhalo

**Comment by barryhalo** [comment]
> HDF 154-157 looks good to me, so approving!
> 
> Edit: I've opened this PR so GH won't let me approve it :D
> Consider +1 to the approve count @barryhalo

Thanks @schystz!

**Comment by tracyarciaga** [comment]
Appreciate your effort on this @schystz

**Pull request merged**